### PR TITLE
Olwa pr

### PR DESCRIPTION
**Pull request by oliver-walker-hms**

### CODE REVIEW

File: src/com/hms_networks/americas/sc/taginfo/TagInfoManager.java [hunk 1]
```diff
@@ -379,6 +379,20 @@ public static synchronized List getTagInfoListFiltered(TagGroup tagGroup) {
     return getTagInfoListFiltered(new TagGroup[] {tagGroup});
   }
 
+  public static ArrayList getRealTimeTagInfoListFiltered(TagGroup tagGroup) {
```
Comment (alexjhawk) [inline src/com/hms_networks/americas/sc/taginfo/TagInfoManager.java:382]: missing javadoc for this method

**Review by alexjhawk — CHANGES_REQUESTED** [review]
A commit that updates the javadocs in the docs/ folder also should be added, as the javadocs in that folder are available for public view at https://hms-networks.github.io/sc-flexy-tag-info-lib/index.html

**Comment by alexjhawk** [comment]
It also looks like you need to rebase this branch with the latest changes

**Review by TomKimsey — COMMENTED** [review]
Why is realtime log getting added? Are you using it for something in the project you are working on?

**Comment by TomKimsey** [comment]
> Why is realtime log getting added? Are you using it for something in the project you are working on?

I can see why standard connectors may need some mechanism to configure the polling rate, but the project you are working on has polling rates set in the config file. Having the realtime log enabled will just use resources.

File: src/com/hms_networks/americas/sc/taginfo/TagInfoManager.java [hunk 2]
```diff
@@ -1,15 +1,16 @@
 package com.hms_networks.americas.sc.taginfo;
 
+import java.io.IOException;
```
Comment (TomKimsey) [inline src/com/hms_networks/americas/sc/taginfo/TagInfoManager.java:3]: Commit message spelling error : organize

**Review by TomKimsey — CHANGES_REQUESTED** [review]

**Review by TomKimsey — APPROVED** [review]

**Review by alexjhawk — APPROVED** [review]

**Pull request merged**